.circleci/test_xrt.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
```

Comment:
one day we should get rid of this `.circleci` dir and move it to `.github` or something.

Reply:
oh we already have the `.github/workflows`, maybe we can move things over in a separate pr.

Reply:
Yeah, let's move this in a follow-up PR